shared/testutil/helpers_test.go

```diff
@@ -12,6 +12,42 @@ import (
 	"github.com/prysmaticlabs/prysm/shared/params"
 )
 
+func TestSetupInitialDeposits(t *testing.T) {
+	deposits, privKeys := SetupInitialDeposits(t, 1)
+	if len(deposits) != 1 {
+		t.Fatal("incorrect number of deposits returned")
```

Comment:
can you also log the length of deposits in the error message ? ex: 
`incorrect number of deposits returned, wanted 1 but got 2` 

Follow the same pattern for the other errors

Reply:
Updated.